libnetdata/string/utf8.c

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "../libnetdata.h"
+
+/**
+ * Get byte length
+ *
+ * Measure the utf8 length
+ *
+ * @param c is the utf8 character
+ *  *
+ * @return It reurns the length of the specific character.
+ */
+char url_utf8_get_byte_length(char c) {
+    if(!IS_UTF8_BYTE(c))
+        return 1;
+
+    char length = 0;
+    while(likely(c & 0x80)) {
```

Comment:
Why don't you use `IS_UTF8_BYTE()` here?

Reply:
This function was an old contribution that we received, I only moved it from `libnetdata/url/url.c` to here, because it became a common code for other files. I agree with your suggestion and I changed it. Thanks!